packages/dashboard-core-plugins/src/linker/LinkerOverlayContent.tsx

```diff
@@ -48,6 +50,11 @@ export type LinkerOverlayContentProps = {
 export type LinkerOverlayContentState = {
   mouseX?: number;
   mouseY?: number;
+  dialogX?: number;
+  dialogY?: number;
+  offsetX: number;
+  offsetY: number;
```

Comment:
Since these are all in pairs, one could define them all as points or something, like:
```
type Point = { x: number, y: number }

export type LinkerOverlayContentState = {
  mouse?: Point;
  dialog?: Point;
  offset?: Point;
};
```
Might make it a little bit cleaner. Of course, not a big deal.